### ISSUE TITLE
TypeError: SyncBucket.__init__() got an unexpected keyword argument 'file_size_limit'

### ISSUE THREAD

**Issue by kevinschaich** [supabase-community/supabase-py#392]
**Describe the bug**

Getting the following error when attempting to download a file from a storage bucket:

```
TypeError: SyncBucket.__init__() got an unexpected keyword argument 'file_size_limit'
```

**To Reproduce**

```python
url: str = os.environ.get("PROD_SUPABASE_URL")
key: str = os.environ.get("PROD_SUPABASE_KEY") 
supabase = create_client(url, key)
file = supabase.storage().get_bucket('raw-data').list()
```

**Expected behavior**

File downloads successfully

**Comment by kevinschaich** [comment]
@anand2312 I'm on `1.0.1` (latest) – looks like that's the latest release on GitHub / [PyPi](https://pypi.org/project/supabase/). Really caught this one at a bad time 😆 

You guys planning on releasing 1.0.2 with the fix soon?

**Comment by anand2312** [comment]
@kevinschaich I think so, yeah. If it's urgent you can install from github, the `develop` branch has the storage-py dependency updated

**Comment by anand2312** [comment]
Fixed with release of 1.0.2
